docker-compose.yml

```
@@ -211,18 +235,18 @@ services:
       - peregrine-service
       - sheepdog-service
   jupyter-service:
-    image: "quay.io/cdis/jupyter-slim:jupyter-covid1.1.3"
+    image: "quay.io/cdis/jupyter-slim:jupyter-covid1.1.12"
```

```suggestion
    image: "quay.io/cdis/jupyter-slim:latest"
```
suggesting `latest` so we don't pin on a PRC tag

---

👍